### ISSUE TITLE
STOP logging via intent hyperlink

### ISSUE THREAD

**Issue by jkdoshi** [mendhak/gpslogger#1103]
(Submitting this one separately from the previous one, as they are different)

Thank you for the (nearly) perfect app!

I am launching it from a webapp using the `gpslogger://properties/...` hyperlink. And my server returns a profile that sends POST to my webapp's custom URL. So now the webapp can track the phone's GPS. Perfect!

One imperfection :-(

Testing the app and looking at all documentation, I couldn't find this --- Upon clicking the intent hyperlink, the web app is able to auto-START logging in my dynamically generated profile. But how can it trigger STOP logging? Basically I want to provide one hyperlink to START logging and another to STOP logging. The first one (START) is working just fine (`startonapplaunch` in properties file), but the second one (STOP) I don't know how to do.

Is there a way? A hint either in the intent hyperlink or in the dynamically generated properties file (profile) would be very helpful.

**Comment by jkdoshi** [comment]
Basically, what I'm asking is how to send `immediatestop=true` via `gpslogger://properties/...` hyperlink (or inside the profile properties file).

**Comment by BorderCounter8812** [comment]
i have the same problem with my oneplus 7t, but only since the update to android12. have you found a solution to the problem?

**Comment by BorderCounter8812** [comment]
addendum to my previous post:
if i have logging via komoot active at the same time, the problem does not occur. not even with this app.

**Comment by mendhak** [comment]
@WilliamRelken has done a pull request for this which is great, I've added it to milestone 130.

OK if I understood correctly you're basically using the .properties file to automate 'starting' and 'stopping' the logging? It's not the  intention of this .properties file, it's only meant to be a profile loader and the automation is an unintended side effect, but I've accepted the PR which adds the stop on launch bit since it's adding a minimum overhead. 

To be clear you're only able to do this via the .properties file is that right, I didn't think that sending actual querystring parameters would work.
